.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release to GitHub Container Registry
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Build Docker image
+      run: docker build -t model-serve .
+
+    - name: Log into GitHub Container Registry
+      run: echo "${{ secrets.TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
+
+    - name: Tag Docker image
+      run: docker tag model-serve ghcr.io/remla2024-team14/model-serving:latest
```

Comment:
You could define a variable to store the IMG which can be less error prone:
`IMG=ghcr.io/remla2024-team14/model-serving:latest`